documentation/topics/constraints.md

```diff
@@ -0,0 +1,188 @@
+# Constraints
+
+Constraints are a way of validating an input type. This validation can be used in both attributes and arguments. The kinds of constraints you can apply depends on the type the data. You can find all types in m:Ash.Type . Each type has its own page on which the available constraints are listed. For example in m:Ash.Type.String you can find 5 constraints:
```

Comment:
backticks for module, `Ash.Type.String`

Reply:
addressed [bb4612c](https://github.com/ash-project/ash/pull/514/commits/bb4612c8519ebd737044fba86b3fd6ebf92ee9a3)